ansys/api/modelcenter/v0/grpc_modelcenter_workflow.proto

```diff
@@ -833,6 +833,14 @@ service ModelCenterWorkflowService {
   rpc ReferenceArrayGetLength(ansys.api.modelcenter.v0.element_messages.ElementId)
     returns (ansys.api.modelcenter.v0.variable_value_messages.IntegerValue);
 
+  /**
+   * Set the length of a reference array.
+   * If larger that the current size, the array will be expanded with empty elements.
+   * If smaller than the current size, items at the end of the array will be dropped.
```

Comment:
May want to document what error you'll get back if you pass an invalid (e.g. negative) size (`OUT_OF_RANGE` seems reasonable to me).